automated install/basic-install.sh

```diff
@@ -1388,27 +1406,61 @@ check_service_active() {
   fi
 }
 
-# Systemd-resolved's DNSStubListener and dnsmasq can't share port 53. 
+# Systemd-resolved's DNSStubListener and dnsmasq can't share port 53.
 disable_resolved_stublistener() {
-  echo -en "  ${INFO} Testing if systemd-resolved is enabled"
-  # Check if Systemd-resolved's DNSStubListener is enabled and active on port 53
-  if check_service_active "systemd-resolved"; then
-    # Check if DNSStubListener is enabled
-    echo -en "  ${OVER}  ${INFO} Testing if systemd-resolved DNSStub-Listener is active"
-    if ( grep -E '#?DNSStubListener=yes' /etc/systemd/resolved.conf &> /dev/null ); then
-      # Disable the DNSStubListener to unbind it from port 53
-      # Note that this breaks dns functionality on host until dnsmasq/ftl are up and running
-      echo -en "${OVER}  ${TICK} Disabling systemd-resolved DNSStubListener"
-      # Make a backup of the original /etc/systemd/resolved.conf
+#  echo -en "  ${INFO} Testing if systemd-resolved is enabled"
+  # Check if Systemd-resolved's DNSStubListener flag is present
+  if [[ "$systemd_resolved_flag" -gt 0 ]]; then
+    if check_service_active "systemd-resolved"; then
+      # Check if DNSStubListener is enabled
+      echo -en "  ${OVER}  ${INFO} Testing if systemd-resolved DNSStub-Listener is active"
+      if ( grep -E '#?DNSStubListener=yes' /etc/systemd/resolved.conf &> /dev/null ); then
+        # Disable the DNSStubListener to unbind it from port 53
+        # Note that this breaks dns functionality on host until dnsmasq/ftl are up and running
+        echo -en "${OVER}  ${TICK} Disabling systemd-resolved DNSStubListener"
+        # Make a backup of the original /etc/systemd/resolved.conf
+        # (This will need to be restored on uninstallation)
+        ${SUDO} sed -r -i.orig 's/#?DNSStubListener=yes/DNSStubListener=no/g' /etc/systemd/resolved.conf
+        echo -e " and restarting systemd-resolved"
+        ${SUDO} systemctl reload-or-restart systemd-resolved
+      else
+        echo -e "${OVER}  ${INFO} Systemd-resolved does not need to be restarted"
+      fi
+    fi
+  else
+    echo -e "${OVER}  ${INFO} Systemd-resolved is not enabled"
```

Comment:
You don't need to echo this if you aren't echoing that you're testing for systemd-resolved

Reply:
Forgot to un-comment line 1411 - Fixed.